src/drm/drm_handler.test.js

```diff
@@ -110,8 +110,7 @@ describe('DRMHandler', function() {
       expect(console.log).toHaveBeenNthCalledWith(1,
         LOG_WARN_HEAD_MESSAGE,
         LOG_WARN_STYLE,
-        'No one license server is found. The expected result for this behavior is clear the current license.',
-      )
+        'No one license server is found. The expected result for this behavior is clear the current license.')
```

Comment:
I would phrase as `No license server was found. The expected result for this behaviour is to clear the current license.`

Reply:
Done here: 4d85a7f